webapp/packages/core-blocks/src/FormControls/Radio.tsx

```diff
@@ -184,27 +103,37 @@ export const Radio: RadioType = observer(function Radio({
     checked = state[name] === value;
   }
 
-  return styled(
-    useStyles(
-      baseFormControlStyles,
-      radioStyles,
-      ...(mod || []).map(mod => radioMod[mod]),
-      !ripple && noRippleStyles,
-      rest.disabled && radioState.disabled,
-    ),
-  )(
-    <field className={s(sizeStyles, { ...layoutProps }, className)}>
-      <radio>
-        <input {...rest} type="radio" id={id} name={name} value={value ?? ''} checked={checked} onChange={handleChange} />
-        <radio-background>
-          <radio-outer-circle />
-          <radio-inner-circle />
-        </radio-background>
-        {ripple && <radio-ripple />}
-      </radio>
-      <label {...use({ disabled: rest.disabled })} htmlFor={id}>
+  return (
+    <div className={s(styles, { ...layoutProps, field: true, menu: mod?.includes('menu') }, className)}>
```

Comment:
This will apply class `theme-radio_small` on `.field` in the case when it's a menu (this differs from the original behavior seems like a bug)